docs/setup/settings/servers.md

```diff
@@ -54,7 +54,7 @@ RTP servers operate independently of your server.
 
 For example, if your server is located in London, you can choose to run your media in New York (if, for example, you have customers there).
 
-Each regional zone encompasses multiple servers to provide high availability.
+Each regional zone encompasses various servers to provide high availability.
```

Comment:
Please change "various" to "multiple", thanks

Reply:
Done[](url)